llvm/lib/Target/AMDGPU/FLATInstructions.td

```diff
@@ -535,7 +535,7 @@ multiclass FLAT_Atomic_Pseudo_NO_RTN<
   ValueType vt,
   ValueType data_vt = vt,
   RegisterClass data_rc = vdst_rc,
-  bit isFP = isFloatType<data_vt>.ret,
+  bit isFP = data_vt.isFP,
```

Comment:
Can this be folded into its only use? Or if not, it should be a defvar instead of an argument. Same for the other instances in this file.

Reply:
Done